src/reducers/columnStateHelper.js

```diff
@@ -29,15 +30,10 @@ const DRAG_SCROLL_BUFFER = 100;
  * @return {!Object}
  */
 function initialize(state, props, oldProps) {
-  const { scrollLeft, scrollToColumn } = props;
+  const { scrollLeft } = props;
   let { columnResizingData, isColumnResizing, scrollX } = state;
 
-  if (scrollLeft !== undefined &&
-      (!oldProps || scrollLeft !== oldProps.scrollLeft)) {
-    scrollX = scrollLeft;
-  }
-
-  scrollX = scrollTo(state, props, oldProps.scrollToColumn, scrollX);
+  const columnAnchor = getColumnAnchor(state, props, oldProps);
```

Comment:
columnAnchor decides the marker for the first/column column along it's offset from the viewport boundary